core/src/Cuda/Kokkos_Cuda_Parallel_MDRange.hpp

```diff
@@ -95,11 +95,32 @@ class ParallelFor<FunctorType, Kokkos::MDRangePolicy<Traits...>, Kokkos::Cuda> {
 
   inline void execute() const {
     if (m_rp.m_num_tiles == 0) return;
-    const auto maxblocks = m_rp.space().cuda_device_prop().maxGridSize;
+    const auto maxblocks  = m_rp.space().cuda_device_prop().maxGridSize;
+    const auto maxthreads = m_rp.space().cuda_device_prop().maxThreadsDim;
+    const auto maxThreadsPerBlock =
+        m_rp.space().cuda_device_prop().maxThreadsPerBlock;
```

Comment:
The compiler complains because `maxThreadsPerBlock` is only used in `KOKKOS_ASSERT` and so it is unused in release mode. You can `[[maybe_unused]]` to silence the warning.

Reply:
done